cluster/manifests/prometheus/configmap.yaml

```diff
@@ -53,7 +53,22 @@ data:
       rules:
       - record: job:skipper_count_by_host_code:sum
         expr: sum(rate(skipper_serve_host_count{}[1m])) by (host,code)
+    {{if eq .Cluster.ConfigItems.network_monitoring_enabled "true" }}
```

Comment:
nit: it's easier to see the `if` when it's not indented, similar to https://github.com/zalando-incubator/kubernetes-on-aws/blob/ddcf978e9dfb27851d13f3727adf1264f193ab08/cluster/manifests/prometheus/configmap.yaml#L82-L93